spec/lib/appsignal/hooks/sidekiq_spec.rb

```diff
@@ -1,145 +1,69 @@
 describe Appsignal::Hooks::SidekiqPlugin do
```

Comment:
I can't find a test in this spec file that tests this new scenario?

A display class that is `Model.delay.method` or something